website/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
```

Comment:
To make it easy to copy the commands using the Github's copy button, can we remove the prefixing `$` signs so that it is easier for the user of this developer doc?